PageObjectPattern/src/test/resources/features/apitest/spotifyGetInfo.feature

```diff
@@ -0,0 +1,17 @@
+# Application token is used for authorization
+#
+Feature: Getting  Information by sending API call
```

Comment:
What about "Getting  Information by sending API call" -> "Integration tests for v1/new-releases endpoint"?